pkgs/by-name/sy/sysdig-cli-scanner/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  makeWrapper,
+}:
+let
+  versionMetadata = builtins.fromTOML (builtins.readFile ./sysdig-cli-scanner.versions.toml);
+  fetchForSystem = versionMetadata.${stdenv.system} or (throw "unsupported system ${stdenv.system}");
```

Comment:
Haven't come across this pattern; very interesting approach!

Reply:
Thank you!